src/modules/todos.js

```diff
@@ -12,16 +12,17 @@ const renderTodoList = () => {
 
   const updatedTodos = clearCompleted(todos);
 
-  updatedTodos.forEach((todo) => {
+  updatedTodos.forEach((todo, index) => {
     const todosListItem = document.createElement('li');
     todosListItem.classList.add('task-list-item');
+    todosListItem.setAttribute('draggable', true);
+    todosListItem.dataset.index = index;
 
     const checkboxWrapper = document.createElement('label');
     checkboxWrapper.classList.add('checkbox-wrapper');
 
     const checkbox = document.createElement('input');
     checkbox.classList.add('checkbox');
```

Comment:
- [ ] Well done :clap: you did a very nice job :+1: but there are some problems with your project when I click on the checkbox and I check local storage it duplicates the data and when I refresh the screen the checked item is deleted automatically so kindly fix this issue for more details check my video :slightly_smiling_face: 

[screen-capture (9).webm](https://github.com/SiveMdluli/To-Do-List/assets/45226848/880fcba6-9c8f-4940-82b7-3f31faeb2a98)